web/js/layers/wv.layers.modal.js

```diff
@@ -62,14 +62,23 @@ wv.layers.modal = wv.layers.modal || function(models, ui, config) {
     $(window).resize(resize);
   };
 
-  //Create container for 'by interest' filters buttons
+  // Create container for 'by interest' filters buttons
   var $nav = $('<nav></nav>').attr('id', 'categories-nav');
 
-  //Create container for breadcrumb
+  // Create container for breadcrumb
   var $breadcrumb = $('<nav></nav>').attr('id', 'category-breadcrumb');
 
-  var checkSettings = function(current, source) {
-    _.each( source.settings, function( setting ) {
+  /**
+   * var checkMeasurementSettings - Checks the (current) measurement's source
+   *  for a setting and returns true if present.
+   *
+   * @param  {type} current The current config.measurements measurement.
```

Comment:
@ZachTRice `type` should be replaced by parameter object type `String, boolean, Array, etc`
```
* @param {Object} source The current measurement source.
```
Also looks like the `current` parameter isn't used